stable/jenkins/values.yaml

```diff
@@ -47,9 +47,10 @@ Master:
   # NodePort: <to set explicitly, choose port between 30000-32767
   ContainerPort: 8080
   # Enable Kubernetes Liveness and Readiness Probes
+  # ~ 2 minutes to allow Jenkins to restart when upgrading plugins. Set ReadinessTimeout to be shorter than LivenessTimeout.
   HealthProbes: true
-  HealthProbesTimeout: 60
-  # ~2 minutes to allow Jenkins to restart when upgrading plugins
+  HealthProbesTimeout: 90
```

Comment:
For consistency, you should rename `HealthProbesTimeout` to `HealthProbesLivenessTimeout`.